docs/semgrep-code/getting-started.md

```diff
@@ -29,333 +28,146 @@ import PlatformSigninGithub from "/src/components/procedure/_platform-signin-git
 import PlatformSigninGitlab from "/src/components/procedure/_platform-signin-gitlab.md"
 import PlatformDetectGhRepos from "/src/components/procedure/_platform-detect-ghrepos.md"
 
-# Getting started with Semgrep Code
-
-Secure your code quickly and continuously by scanning with Semgrep Code, a fast and lightweight SAST (Static Application Security Testing) engine that leverages Semgrep OSS.
-
-<SemgrepScan />
-
-Semgrep Code is transparent: you can fully configure what rules are run and inspect the Semgrep syntax to understand how the finding was detected. The content of a rule can be customized to improve the true positive rate of a rule or its message to fellow developers.
-
-This document provides steps to get started with Semgrep Code for all tiers except where indicated.
-
-## Semgrep Code with Semgrep Cloud Platform
-
-Semgrep Code includes Semgrep Cloud Platform (SCP), a web application that helps security teams meet their goals, such as:
-
-* Sorting, filtering, triaging, and remediating security issues.
-* Enforcing coding standards through the creation of custom rules.
-* Preventing insecure code from reaching production or staging servers by blocking pull or merge requests, based on how you configure your rules.
-* Notifying security teams of findings (results) as well as communicating with other developers by leaving pull or merge request comments in GitHub, GitLab, or BitBucket.
-
-:::tip Suggested workflows 
-* You can use Semgrep Cloud Platform to scan remote repositories (GitHub, GitLab, or BitBucket) and consolidate the findings.
-* You can also use Semgrep Cloud Platform to consolidate findings from a Semgrep CLI scan performed on a **local** machine.
-* Semgrep Code can be integrated into your custom infrastructure without SCP. See [API](/semgrep-cloud-platform/semgrep-api) for details.
-:::
-
-### Scanning a repository
-
-#### Signing in to Semgrep Cloud Platform
-
-<PlatformSigninIntro />
+# Scan your codebase
 
-<Tabs
-    defaultValue="signin-github"
-    values={[
-    {label: 'Sign in with GitHub', value: 'signin-github'},
-    {label: 'Sign in with GitLab', value: 'signin-gitlab'},
-    ]}
->
+Semgrep Code is a static application security testing (SAST) tool that scans your repository for security vulnerabilities. You can scan your repository by integrating Code into your CI/CD pipeline, or you can scan repositories available locally on your machine.
 
-<TabItem value='signin-github'>
+## Use the Semgrep Cloud Platform (SCP) to automate the continuous scanning of your code
```

Comment:
This is a pretty tricky section. Right now, the steps as-is are not complete. If the user is coming from the quickstart, can you confirm that the new onboarding **includes adding an SCM connection?** If not, we'll need an onboarding document to cover that, so that we can keep **this** doc focused on running the code product specifically (something like `semgrep ci --code` and not just the generic `semgrep ci` command, which will run a code and SCA scan by default.